### PR TITLE
Add check for openshift subscription

### PR DESCRIPTION
**Pull request by nmars**
Added code to parse the `operators.openshift.io/valid-subscription`  annotation from the CSV and add this information to the `parsed_operator_data.yml` artifact.

### CODE REVIEW

File: roles/parse_operator_bundle/templates/parsed_operator_data.yml.j2 [hunk 1]
```diff
@@ -13,6 +13,7 @@ operator_allnamespaces_support: {{ operator_allnamespaces_support }}
 operator_ownnamespace_support: {{ operator_ownnamespace_support }}
 operator_singlenamespace_support: {{ operator_singlenamespace_support }}
 operator_multinamespace_support: {{ operator_multinamespace_support }}
+operator_valid_subscription: {{ operator_valid_subscription }}
```
Comment (lslebodn) [inline roles/parse_operator_bundle/templates/parsed_operator_data.yml.j2:16]: Have you consider omitting `operator_valid_subscription` the same as for `crd_paths` few lines after?
IMHO relying on default value "Not found'" is not ideal.

Comment (nmars) [reply roles/parse_operator_bundle/templates/parsed_operator_data.yml.j2]: Good suggestion

**Comment by dirgim** [comment]
LGTM

**Review by dirgim — APPROVED** [review]

File: roles/parse_operator_bundle/templates/parsed_operator_data.yml.j2 (hunk 1 above)
Comment (lslebodn) [reply roles/parse_operator_bundle/templates/parsed_operator_data.yml.j2]: thank you for changes

**Pull request merged**